# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/fastfetch/t2-logo.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---


--- fastfetch-2.32.1/src/logo/ascii/t2.txt.vanilla	2024-12-19 18:04:30.169688876 +0100
+++ fastfetch-2.32.1/src/logo/ascii/t2.txt	2024-12-19 18:05:38.459690187 +0100
@@ -1,6 +1,15 @@
-${c2}TTTTTTTTTT
-    tt   ${c1}222${c2}
-    tt  ${c1}2   2${c2}
-    tt     ${c1}2${c2}
-    tt    ${c1}2${c2}
-    tt  ${c1}22222${c2}
\ No newline at end of file
+$3
+                        //$2
+                       $3//$2
+ $1TTTTTTTTTTTTTTT$2      $3//$2 $1..::$2  $4:-:.$2
+       $4ttt$2           $3//$2 $1.::.$2    $4.:-:$2
+       $4ttt$2          $3//$2  $1.:.$2       $4--$2
+       $4ttt$2         $3//$2            $4:-:$2
+       $4ttt$2        $3//$2           $4.:-:$2
+       $4ttt$2       $3//$2           $4:-:.$2
+       $4ttt$2      $3//$2          $4.--.$2
+       $4ttt$2     $3//$2         $4.:-:$2
+       $4ttt$2    $3//$2          $4...$2
+       $4ttt$2   $3//$2
+            $3//$2         $3.:::::::::::.$2
+           $3//$2
--- fastfetch-2.11.0/src/logo/builtin.c.vanilla	2024-05-02 11:03:36.141344213 +0200
+++ fastfetch-2.11.0/src/logo/builtin.c	2024-05-02 11:05:01.616041875 +0200
@@ -3918,7 +3918,9 @@
         .names = {"T2", "T2 SDE"},
         .lines = FASTFETCH_DATATEXT_LOGO_T2,
         .colors = {
+            FF_COLOR_FG_RED,
             FF_COLOR_FG_WHITE,
+            FF_COLOR_FG_YELLOW,
             FF_COLOR_FG_BLUE,
         },
     },
